### DISCUSSION TITLE
Web Builder v0.9 and MKS-TinyBee 1.0

### DISCUSSION THREAD

**Discussion by LEOOZERAN** [grblHAL/core#434] [Q&A]
In the Web Builder v0.9f program, the MKS-TinyBee 1.0 board firmware does not work. FOR a simple ESP32. In the ioSender program. Reports a board error (it doesn’t see it) In other programs, there’s also an error (it doesn’t see the board)

**Comment by terjeio** [comment]
Duplicate of #433, closed.
